cmd/kubeadm/app/cmd/certs.go

```diff
@@ -441,7 +441,7 @@ func newCmdCertsExpiration(out io.Writer, kdir string) *cobra.Command {
 
 					s := fmt.Sprintf("%s\t%s\t%s\t%-8v",
 						e.Name,
-						e.ExpirationDate.Format("Jan 02, 2006 15:04 MST"),
+						e.ExpirationDate.UTC().Format("Jan 02, 2006 15:04 MST"),
```

Comment:
these changes in certs expiration command for kubeadm seem fine.
but we should only accepted them if kubectl output is made consistent.

Reply:
you're right.
This is safe, it is just a time zone that changes the output time

Reply:
before:
![image](https://user-images.githubusercontent.com/55286565/120217123-79c1d480-c26a-11eb-9a5a-511ae6e56d33.png)

after:
![image](https://user-images.githubusercontent.com/55286565/120217189-978f3980-c26a-11eb-91a5-999e7334b324.png)